test/extensions/filters/http/cache/cache_filter_integration_test.cc

```diff
@@ -108,9 +108,14 @@ class CacheIntegrationTest : public Event::TestUsingSimulatedTime,
   OptRef<const Http::TestResponseTrailerMapImpl> empty_trailers_;
 };
 
-INSTANTIATE_TEST_SUITE_P(Protocols, CacheIntegrationTest,
-                         testing::ValuesIn(HttpProtocolIntegrationTest::getProtocolTestParams()),
-                         HttpProtocolIntegrationTest::protocolTestParamsToString);
+// Test for HTTP1/2 only.
+// TODO(asingh-g): Fix test suite for HTTP/3.
```

Comment:
Can you file a GH issue for the tests which don't support HTTP/3? Out of curiosity, do you understand why they fail? (Presumably they're all unrelated to the work your doing so maybe it's not obvious why they're broken)